conf/test.config

```diff
@@ -13,56 +13,42 @@ params {
   
   // Limit resources so that this can run on GitHub Actions
   max_cpus = 2
-  max_memory = 6.GB
+  max_memory = 4.GB
```

Comment:
Why 4GB and not 6?

Reply:
Actually, there was no particular reason for that. I will set it to 6 back :)

Reply:
That was left over from testing on my virtual machine with only 4GB of memory :-)